target/linux/ipq40xx/image/Makefile

```diff
@@ -332,6 +332,21 @@ define Device/engenius_emd1
 endef
 TARGET_DEVICES += engenius_emd1
 
+define Device/engenius_enh1350ext
+	$(call Device/FitImage)
+	DEVICE_VENDOR := EnGenius
+	DEVICE_MODEL := ENH1350EXT
+	DEVICE_DTS_CONFIG := config@4
+	BOARD_NAME := enh1350ext
```

Comment:
Can you tell me what's the obsession with BOARD_NAME? I had to remove this time and time again, so what gives?

Reply:
I'd say it's just a copy/paste thing, as many old devices used as blueprint still have it.
I similarly have to remove SUPPORTED_DEVICES in a little less than half of the device support PRs I merge, because many just copy that over as well from existing devices that had a rename, and then even creatively change the string there ...